### PR TITLE
Update last page to refer to GCP

### PR DESCRIPTION
**Pull request by rigoford**
Last page erroneously refers to VirtualBox (BOSH Lite) and not GCP

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -635,13 +635,14 @@ <h4 class="terminal-code-text">$ bosh -d learn-bosh-on-gcp logs router/0</h4>
     
     <div class="section last"><div class="center-column"><h1>Done!</h1>
 
-<p>In this tutorial we used BOSH to deploy services, updated our deployment with source changes, scaled the number of services and changed their properties. We recovered from failing service, failing VM and failing deploy.</p>
+<p>In this tutorial we created a BOSH Bastion in Google Cloud Platform and used the BOSH CLI on that bastion to create a BOSH Director. We deployed services, updated our deployment with source changes, scaled the number of services and changed their properties on that BOSH Director. We also recovered from a failing service, failing VM and failing deploy.</p>
```
Comment (cagiti) [inline index.html]: We deployed a release

Comment (cagiti) [inline index.html]: We also recovered a failing service, failing VM and a failing deploy.

File: index.html [hunk 2]
```diff
@@ -635,13 +635,14 @@ <h4 class="terminal-code-text">$ bosh -d learn-bosh-on-gcp logs router/0</h4>
...
 
-<p>We were using VirtualBox environment with Warden CPI (Cloud Provider Interface). Warden is a Linux container management tool and Warden CPI abstracts VMs as Linux containers. The Director can work with any CPI that implements a certain API to manage IaaS resources. There are several supported CPIs for different IaaS providers: AWS, GCP, Openstack, vSphere and vCloud. Read more about CPIs here: <a href="http://bosh.io/docs/cpi-api-v1.html">http://bosh.io/docs/cpi-api-v1.html</a>.</p>
+<p>The BOSH Director can work with any CPI (Cloud Provider Interface) that implements a certain API to manage IaaS resources. There are several supported CPIs for different IaaS providers: AWS, GCP, Openstack, vSphere, vCloud and VirtualBox (a.k.a. BOSH Lite). YOu can read more about CPIs here: <a href="http://bosh.io/docs/cpi-api-v1.html">http://bosh.io/docs/cpi-api-v1.html</a>.</p>
```
Comment (cagiti) [inline index.html]: You

**Review by cagiti — CHANGES_REQUESTED** [review]

**Comment by rigoford** [comment]
@cagiti updates made. Please review again.

**Review by cagiti — APPROVED** [review]

**Pull request merged**